### ISSUE TITLE
Performance?

### ISSUE THREAD

**Issue by shner-elmo** [jweinst1/pysimd#2]
Nice project, would be nice if you posted some benchmarks so potential users can see the performance benefits of using this over pure Python solutions, or even compared to pandas `Series`, thanks

**Comment by jneb** [comment]
Absolutely! Since the overhead of calling will be relatively long, it is hard to decide when to use this project.
A comparison with pandas and numpy should be the least to expect, but I would also like to see it compares with numba and intel-numpy.
The must important figure probably is: at what size of vector is this going to be significantly faster?
